Dockerfile.sydney

```diff
@@ -19,7 +19,7 @@ WORKDIR /usr/src/app
 
 COPY . .
 
```

Comment:
I'm worried that it might be too late later to rename "Golden Gate" Can we use GGX Chain where possible?